spec/support/user_fixture.rb

```diff
@@ -6,13 +6,16 @@ def self.reset_all_users
   end
 
   def self.normal_user
-    password = "thi$ 1s cOmplExEr"
-    user = User.new(first_name: "Joe", last_name: "Schmoe",
-                    email: "joe@schmoe.com", password: password, password_confirmation: password)
-    def user.clear_password
-      "thi$ 1s cOmplExEr"
+    password = 'thi$ 1s cOmplExEr'
+    User.create!(first_name: 'Joe', last_name: 'Schmoe', email: 'joe@schmoe.com',
```

Comment:
Rubocop

```
spec/support/user_fixture.rb:10:30: C: Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
    User.create!(first_name: 'Joe', last_name: 'Schmoe', email: 'joe@schmoe.com',
```